packages/aws-rfdk/lib/deadline/lib/render-queue-ref.ts

```diff
@@ -211,6 +212,20 @@ export interface RenderQueueAccessLogProps {
   readonly prefix?: string;
 }
 
+/**
+ * Options for security groups of the Render Queue.
+ */
+export interface RenderQueueSecurityGroupsOptions {
+  /**
+   * The security group for the backend components of the Render Queue, which consists of the AutoScalingGroup for the Deadline RCS.
+   */
+  readonly backend?: ISecurityGroup;
+  /**
+   * The security group for the frontend of the Render Queue, which is its load balancer.
+   */
+  readonly frontend?: ISecurityGroup;
+}
```

Comment:
Now it is not options any more. I can see possible two security groups that should be added.
Is any reason we are limited with only one security group for backend and frontend?

Reply:
It's mostly to follow the pattern that CDK has established to take in a [single security group in the constructor](https://github.com/aws/aws-cdk/blob/ea91aa31db9e2f31c734ad6d7e1f64d5d432dfd4/packages/%40aws-cdk/aws-ec2/lib/instance.ts#L122) and have a [method to add more](https://github.com/aws/aws-cdk/blob/ea91aa31db9e2f31c734ad6d7e1f64d5d432dfd4/packages/%40aws-cdk/aws-ec2/lib/instance.ts#L418), as Daniel pointed out earlier.